src/Concordium/Client/Output.hs

```diff
@@ -634,12 +634,12 @@ showRejectReason verbose = \case
       "account does not exist"
   Types.InvalidInitMethod m name ->
     if verbose then
-      printf "invalid init method '%s' of module '%s'" (show name) (show m)
+      printf "invalid init method %s of module '%s'" (show name) (show m)
```

Comment:
`printf` + `show` already puts `""` around the name, so we can remove `'`.

Reply:
Ok, but that is a bug since we use single quotes elsewhere and we should be consistent.

This should have single quotes only, and we should not be using show to avoid double quotes.

Reply:
Can you fix that as well?